src/style.css

```
@@ -1 +1,65 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+:root {
+  --font-color: #00acdc;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 #0000000d;
```

it's incorrect hex color
```suggestion
  box-shadow: 0 2px 4px 0 #0000000d;
```